src/003_example.md

```diff
@@ -482,6 +482,26 @@ attached (dis)proofs:
       Client_auth_injective (all-traces): verified (15 steps)
       Client_session_key_honest_setup (exists-trace): verified (5 steps)
 
+
+### Quit on Warning
+
+As referred to in ["Graphical User Interface"](#sec:gui), in larger models, it's
+very easy to miss wellformedness errors (when writing the Tamarin file, and when
+running the `tamarin-prover`), especially as a long list of pre-computation
+output-text flies past on the terminal: in many cases, the web-server starts up
+correctly, making it harder to notice that something's not right either in a
+rule or lemma.
+
+To ensure that your provided `.spthy` file is free of any errors or warnings
+(and to halt pre-processing and other computation in the case of errors), it can
+be a good idea to use the `--quit-on-warning` flag at the command line. E.g.
```

Comment:
nit: E.g.,

Reply:
>Style guides do not agree on whether or not a comma should follow both these abbreviations. They do all agree that a comma precedes i.e. when the i.e. phrase occurs in a running text (i.e., not enclosed in parenthesis).
>The consensus seems to be in favor of the comma in American usage; against it in British usage.

:-P

Reply:
...but, for consistency with the rest of the document's sake, I will change it :-)